src/backend/base/langflow/components/vectorstores/Qdrant.py

```diff
@@ -20,8 +20,7 @@
 class QdrantVectorStoreComponent(LCVectorStoreComponent):
     display_name = "Qdrant"
     description = "Qdrant Vector Store with search capabilities"
-    documentation = "https://python.langchain.com/docs/modules/data_connection/vectorstores/integrations/qdrant"
-    name = "Qdrant"
+    documentation = "<link url='https://python.langchain.com/docs/modules/data_connection/vectorstores/integrations/qdrant'>Webpage description not available.</link>"
```

Comment:
This link won't work. You gotta put just the URL.

Reply:
done